Typos in Install Beacon agent doc (#6319)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/console/estate/agent/install-agent.mdx

export BEACON_AGENT_PROJECT_ID=<your-project-id>
```

> Running the `beacon-agent setup` command creates a configuration file in the Beacon configuration directory using those environment variables.

Reviewer: ```suggestion
These environment variables are used when you run the `beacon-agent setup` command to create a configuration file in the Beacon configuration directory.
```
